modules/xresources.nix

```
@@ -75,11 +75,18 @@ in {
         <filename>~/.Xresources</filename> link is produced.
       '';
     };
+
+    xresources.path = mkOption {
+      type = types.str;
+      default = "${config.home.homeDirectory}/.Xresources";
+      defaultText = "$HOME/.Xresources";
+      description = "Path to generate the .Xresources file to.";
```

nit: the sentence is a bit awkward as-is. Maybe something like this instead?

```suggestion
      description = "The place to put the generated .Xresources file.";
```

---

IMO, that also sounds a bit awkward. How does this sound instead? (based on https://github.com/nix-community/home-manager/blob/2aeaf65e8f9219c1acdb47bcf278983b3170a344/modules/xsession.nix)
```suggestion
      description = "Path where Home Manager should write the .Xresources file";
```

---

Ah, that sounds a lot better. Let's go with what you suggested.

---

I committed it, with a slight change in my last comment from `write` to `link` just to stay a bit more consistent with the rest of the module's docs.